### PR TITLE
add tasks by state widget

### PR DESCRIPTION
**Pull request by meirwah**
Don't merge till tasks PR merged

### CODE REVIEW

**Comment by meirwah** [comment]
@Bars92 plz review

**Comment by meirwah** [comment]
@Bars92 plz review

File: Widgets/widget-TasksByState.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "id": "tasks-state",
+  "version": -1,
+  "modified": "2018-01-14T22:59:42.479686738Z",
+  "name": "Tasks By State",
+  "dataType": "tasks",
+  "widgetType": "column",
+  "query": "replaced:f" ,
```
Comment (Bars92) [inline Widgets/widget-TasksByState.json]: @aviadl is `replaced:f` means only active tasks?

Comment (aviadl) [reply Widgets/widget-TasksByState.json]: Yes

**Pull request merged**